File: 2017/Rust/aoc_2017/src/day_3.rs
use std::fs;

pub fn day_3_main() {
    //Ingest data
    let file_name = "inputs/day_3_input.txt";
    let contents = fs::read_to_string(file_name)
        .expect("Failed to read File");

    //Split by each line
    let parts: Vec<&str> = contents.split("\n").collect();

    println!("The answer the Part 1 is: {}", part_1(parts.clone()));
    println!("The answer the Part 2 is: {}", part_2(parts.clone()));
}

fn part_1(parts: Vec<&str>) -> i32 {
    let mut sumr = 0;

    return sumr;
}

fn part_2(parts: Vec<&str>) -> i32 {
    let mut sumr = 0;
    return sumr;
}

File: 2023/Python/day_3.py
def parse_input():
    splitter = []
    with open("Inputs/day_3_input.txt") as input_file:
        for line in input_file:
            ltext = line.replace("\n", "")
            ltext = list(ltext)
            splitter.append(ltext)
    return splitter


def part_1():
    sumr = 0
    data = parse_input()

    for i in range(len(data)):
        print(data[i])
    print("The sum of the possible games for Part 1 is: " + str(sumr))


def part_2():
    data = parse_input()
    sumr = 0
    print("The power of all the games in Part 2 is: " + str(sumr))


if __name__ == "__main__":
    part_1()
    # part_2()
